package.json

```diff
@@ -63,8 +69,11 @@
     "dist"
   ],
   "ava": {
+    "extensions": [
+			"ts"
```

Comment:
we should add prettier in a follow-up PR (we could use this repo's usage of prettier as a template https://github.com/transitive-bullshit/react-static-tweets)

Reply:
I can add prettier to this one if you would like